.eslintrc.json

```diff
@@ -1,17 +1,6 @@
 {
-    "extends": "wikimedia",
-    "parserOptions": {
-        "ecmaVersion": 3
```

Comment:
Dropping ES3 compat. makes this a breaking change; worth retaining?

Reply:
Could be done as separate change, but I support dropping it either way, and if/when we do make behaviour changes again, we'd release those with semver-major, of course.